packaging/ubuntu-trusty/debian/rules

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/make -f
+# -*- makefile -*-
+
+# Uncomment this to turn on verbose mode.
+export DH_VERBOSE=1
+
+%:
+	dh $@
+
```

Comment:
I added a clean rule.

But not `go clean`, I don't see why that would be needed.